include/deal.II/lac/scalapack.h

```diff
@@ -290,29 +447,25 @@ class ScaLAPACKMatrix : protected TransposeTable<NumberType>
   * It is assumed that $A$ has full rank: $rank(A) = \min(M,N)$.
   *
   * The following options are supported:
-  * - 1. If <code>transpose==false</code> and $M \geq N$: least squares solution of overdetermined system
```

Comment:
since this is a separate stand-alone commit, can you please cherry-pick it into a stand-alone PR/branch?